Composer/packages/types/src/publish.ts

```diff
@@ -14,9 +14,31 @@ export type PublishResult = {
   eTag?: string;
   log?: string;
   id?: string;
-  time?: Date;
+  time?: string;
   endpointURL?: string;
   status?: number;
+  action?: {
+    href: string;
+    label: string;
+  } | null;
+};
+
+export type ProcessStatus = {
```

Comment:
What's the difference between this and ProvisionStatus defined earlier?

Reply:
Seems like the ProcessStatus is not used. I removed it